brain/compartmentReportReader.cpp

```diff
@@ -178,6 +182,45 @@ std::future<CompartmentReportFrames> CompartmentReportView::load(float start,
 
 std::future<CompartmentReportFrames> CompartmentReportView::loadAll()
 {
+    if (_impl->report->getGIDs().size() == 1)
+    {
+        auto report = _impl->report;
+        auto id = *_impl->report->getGIDs().begin();
+
+        double start = _impl->report->getStartTime();
+        double end = _impl->report->getEndTime();
+        double timestep = _impl->report->getTimestep();
+        auto loadFrameTask = [start, end, timestep, report, id] {
+
+            CompartmentReportFrame frame;
+
+            try
+            {
+                auto data = report->loadNeuron(id);
+                if (data)
+                    frame._impl->data = std::move(*data);
```

Comment:
Indeed, the frame in this case is containing a trace. As you mentioned, this would be changed later.